### ISSUE TITLE
https://www.apkmirror.com

### ISSUE THREAD

**Issue by 3052** [uBlockOrigin/uAssets#25989]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.apkmirror.com/
```

### Description

This page could not be loaded properly due to incorrect / bad filtering rule(s) of adblockers in use. Please disable all adblockers to continue using the website. (click OK if you'd like to learn more)

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.60.0
Firefox: 121
filterset (summary):
 network: 139174
 cosmetic: 49102
 scriptlet: 21366
 html: 2080
listset (total-discarded, last-updated):
 default:
  user-filters: 17-0, never
  ublock-filters: 40903-133, now
  ublock-badware: 12261-6, now
  ublock-privacy: 1408-22, now
  ublock-unbreak: 2562-1, now
  ublock-quick-fixes: 193-6, now
  easylist: 77897-198, now
  easyprivacy: 53249-64, now
  urlhaus-1: 21255-0, 59m
  plowe-0: 3549-1000, now
filterset (user): [array of 35 redacted]
trustedset:
 added: [array of 9 redacted]
switchRuleset:
 removed:
  no-large-media: behind-the-scene false
  no-csp-reports: * true
hostRuleset:
 added: [array of 153 redacted]
 removed:
  behind-the-scene * * noop
  behind-the-scene * image noop
  behind-the-scene * 3p noop
  behind-the-scene * inline-script noop
  behind-the-scene * 1p-script noop
  behind-the-scene * 3p-script noop
  behind-the-scene * 3p-frame noop
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 115209 ms (selfie)
 maxAssetCacheWait: 51 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 17
 network:
  github.com: 17
```
</details>

**Comment by 3052** [comment]
as a workaround - looks like you can click

Don't allow www.apkmirror.com to prompt you again

then click cancel

**Comment by 3052** [comment]
dumb question I am still getting blocked - but the "update now" is grayed out

**Comment by gorhill** [comment]
[Click this link](https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-filters&manual=241110)

**Comment by 3052** [comment]
@gorhill  its an issue with the extension as a whole. I shouldn't have to click a magic link to get an update - it should be surfaces in the GUI

**Comment by gorhill** [comment]
> I shouldn't have to click a magic link to get an update

Filter lists are updated once every ~5 hours in the free CDN servers -- there is no point clicking an "Update now" button when the lists haven't been updated on the CDN servers, this just eats bandwidth without resolving your filter issues.

The "magic link" will bypass CDN servers and fetch directly from GitHub repo, which is not meant to be used as a CDN. If you like this project and want it to continue, you will have to accept this extremely minor inconvenience -- lists update transparently in the background very ~5 hours. For those who can't wait and want an immediate fix, the "magic link" does the trick.

**Comment by 3052** [comment]
@gorhill fair enough, but why not add the link into the extension itself, why put users through this? you could have saved your time and mine

**Comment by stephenhawk8054** [comment]
Next time, just comment in the old thread abd wait for volunteers to reopen the issue. All volunteers are notified with new comments. Don't spam like this.

**Comment by 3052** [comment]
@stephenhawk8054 its their own fault for closing the issue without feedback from the original poster

**Comment by stephenhawk8054** [comment]
Websites are not static forever. The old issues was closed because it was fixed at that time. When the website update again, just continue reporting in that thread.

**Comment by 3052** [comment]
@stephenhawk8054 it was fixed at the GitHub level, but as an end user I have no access to the updated version, unless I click the magic link from @gorhill - which I didn't even know exist until he posted it. its just good manner to wait at least one day or something - otherwise you signal that you dont care about feedback from the poster

**Comment by garry-ut99** [comment]
> 3052: fair enough, but why not add the link into the extension itself, why put users through this? you could have saved your time and mine

- https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches- , especially this link: 
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2958

**Comment by stephenhawk8054** [comment]
Just continue everything in the old thread. If you want to report the website updates again, comment in that thread. If you want to know how to update the filter, you can ask in that thread. No need to open duplicated issue. Same as for original poster. If there's no further comment from original poster, we will close the issue.

We are volunteers of the project, not customer service. If you don't agree how we manage the project, feel free to not use it.

**Comment by gorhill** [comment]
> why not add the link into the extension itself

Because GitHub must not be used as a CDN, it's in their terms of use. It's ok to pull once in a while so we provide a link for those who can't wait. The link will stop working in a few hours, it will do nothing when you click it.

> why put users through this?

For people who can't accept the inconveniences that come with using free CDNs, I strongly suggest they uninstall uBO and find something else. I will not put this project at risk of ceasing to exist just because once in a rare while someone demands an _immediate_ fix.
